client/my-sites/theme/theme-features-card.js

```diff
@@ -18,9 +18,13 @@ const ThemeFeaturesCard = ( { isWpcomTheme, siteSlug, features, translate } ) =>
 			<Card>
 				<ul className="theme__sheet-features-list">
 					{ features.map( ( { name, slug, term } ) => (
-						<li key={ 'theme-features-item-' + slug }>
+						<li
+							key={ 'theme-features-item-' + slug }
+							role="presentation"
+							onClick={ () => onClick?.( slug ) }
+						>
 							{ ! isWpcomTheme ? (
-								<a>{ name }</a>
+								<span href="">{ name }</span>
```

Comment:
I'm curious about why we want to change the tag name and add href attribute 🤔

Reply:
Oh, good catch... I forgot to remove it.
The reason I changed it to `<span>` was because:

1. Linter flagged that `<a>` should have the `href` attribute.
2. If we add `href=""` then the `<a>` becomes clickable (cursor pointer) but it's not meant to be clickable.

Let me know if that makes sense!

Reply:
Got it 👍